Source/Renderer/Context.js

```diff
@@ -276,11 +276,15 @@ define([
         this._blendMinmax = !!getExtension(gl, ['EXT_blend_minmax']);
         this._elementIndexUint = !!getExtension(gl, ['OES_element_index_uint']);
         this._depthTexture = !!getExtension(gl, ['WEBGL_depth_texture', 'WEBKIT_WEBGL_depth_texture']);
-        this._textureFloat = !!getExtension(gl, ['OES_texture_float']);
-        this._textureHalfFloat = !!getExtension(gl, ['OES_texture_half_float']);
         this._fragDepth = !!getExtension(gl, ['EXT_frag_depth']);
         this._debugShaders = getExtension(gl, ['WEBGL_debug_shaders']);
 
+        this._textureFloat = !!getExtension(gl, ['OES_texture_float']);
+        this._textureHalfFloat = !!getExtension(gl, ['OES_texture_half_float']);
+
+        this._textureFloatLinear = !!getExtension(gl, ['OES_texture_float_linear']);
```

Comment:
Does anything need to be taken into account for WebGL 1 vs. 2?

Reply:
I'm assuming that the extension is still required for WebGL 2. I couldn't find anything in the spec that says it isn't allowed in WebGL 2; however, I do see the extension for `OES_texture_float_linear` under the WebGL 2 tab at webglreport.com. Also, I don't see the extension for `OES_texture_half_float_linear` in the WebGL 2 tab, but I do see it under the WebGL 1 tab.

Reply:
According to MDN it is both a WebGL 1 and 2 extension.
https://developer.mozilla.org/en-US/docs/Web/API/OES_texture_float_linear

Reply:
And `OES_texture_half_float_linear` is gone under WebGL 2 and it's included with `OES_texture_float_linear`. (Similar to what they did for floating-point color buffer attachments)

Reply:
Is https://developer.mozilla.org/en-US/docs/Web/API/OES_texture_half_float_linear inccorect?